commerce-api/src/main/java/com/liferay/commerce/constants/CommerceDefinitionTermConstants.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.constants;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommerceDefinitionTermConstants {
+
+	public static final String ORDER_DEFINITION_TERMS_CONTRIBUTOR =
+		"CommerceOrderTerms";
+
+	public static final String SUBSCRIPTIONS_DEFINITION_TERMS_CONTRIBUTOR =
```

Comment:
should be `SUBSCRIPTION_DEFINITION_TERMS_CONTRIBUTOR` rather than `SUBSCRIPTIONS_DEFINITION_TERMS_CONTRIBUTOR` (just remove an "S").